### ISSUE TITLE
Broken build on hub.docker.com

### ISSUE THREAD

**Issue by dekhtyarev** [2gis/k8s-handle#95]
Build on hub.docker.com is broken:
```
...
Extracting MarkupSafe-1.1.1-py3.6-linux-x86_64.egg to /usr/local/lib/python3.6/site-packages
Adding MarkupSafe 1.1.1 to easy-install.pth file
Installed /usr/local/lib/python3.6/site-packages/MarkupSafe-1.1.1-py3.6-linux-x86_64.egg
error: urllib3 1.25.2 is installed but urllib3<1.25,>=1.21.1 is required by {'requests'}
Removing intermediate container 898aedf7dc43
The command '/bin/sh -c apk --no-cache add git ca-certificates bash openssl gcc libc-dev libffi-dev openssl-dev make && cd /opt/k8s-handle && python setup.py install && apk del gcc libc-dev libffi-dev openssl-dev' returned a non-zero code: 1
```

Need repair

**Comment by dekhtyarev** [comment]
Release https://github.com/2gis/k8s-handle/releases/tag/0.5.7

**Comment by dekhtyarev** [comment]
Closes by #96 

Thanks @FreakyGranny !
